portal-web/test/functional/com/liferay/portalweb/paths/portlet/pageeditor/PageEditor.path

```diff
@@ -528,7 +528,7 @@
 </tr>
 <tr>
 	<td>FRAGMENT_SIDEBAR_COLLECTION_PANEL_EXPANDED</td>
-	<td>//*[contains(@class,'panel-header-link collapse-icon')][normalize-space()='${key_collectionName}']</td>
+	<td>//*[contains(@class,'mb-3 panel-header panel-header-link collapse-icon collapse-icon-middle show btn btn-unstyled')][normalize-space()='${key_collectionName}']</td>
```

Comment:
This class name could fail in case that the order of the parameters would be different. Instead of putting everything together, you should put something like: contains(@class,'mb-3') and contains(@class,'panel-header") and blablabla. Please modify it, thanks :)

Reply:
Done!!